.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "trailingComma": "all",
+  "printWidth": 90,
+  "singleQuote": false
+}
```

Comment:
Nice, I hope now we don't have  this back and forth between different editor settings 🙈